relativeDistA: 0.25
relativeDistB: 2.00
relativeDistC: 1.40
relativeDistK: 0.75
relativeAngleA: 0.08
relativeAngleB: 3.00
relativeAngleC: rad(pi/2)
relativeAngleK: 1.95
relativeVelA: 0.20 
relativeVelB: 1.20
relativeVelC: 0.00
relativeVelK: 0.05
subgoalAngleA: 0.45
subgoalAngleB: 1.00
subgoalAngleC: 0.00
subgoalAngleK: 0.90
obstacleDistA: 20.0
obstacleDistB: 0.40
obstacleDistK: 0.03
velocityA: 0.30
velocityB: 1.60
velocityC: 1.10
velocityK: 0.5
angularVelA: 0.70
angularVelB: 1.00
angularVelC: 0.00
angularVelK: 0.01
accelA: 0.20
accelB: 1.00
accelC: 0.00
accelK: 0.01